src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1573,8 +1573,14 @@ else if (logger.isDebugEnabled())
                     Keyspace.open(command.ksName).getColumnFamilyStore(command.cfName).metric.blockingReadRepairLatency.addNano(latency);
                 }
             }
+            for (AbstractReadExecutor exc : readExecutors) {
```

Comment:
We should probably do this after each time we receive a row, i.e. in a finally on line 1505 (after we go through and #get each row)